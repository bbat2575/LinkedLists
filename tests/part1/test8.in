NEW +1
NEW -1
NEW a
NEW .
NEW 1.
NEW .1
NEW 1.1
NEW 2e2
NEW 1 2
NEW  3
NEW   4
NEW
NEW 
NEW  
5NEW
6 NEW
NEW7
 NEW 8
  NEW 9
NE W 10
N EW 11
new 12
NeW 13
NWE 14
 NEW
  NEW
