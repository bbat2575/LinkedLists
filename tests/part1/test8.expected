
Type HELP for a list of options
Press <CTRL + D> to Quit

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: NEW

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> INVALID COMMAND: INPUT

> 
